honeybee_vtk/tovtk.py

```diff
@@ -0,0 +1,311 @@
+"""Functions to create vtk objects based on the data extracted from hbjson."""
```

Comment:
I'm a fan of making the module names human readable. I would use `to_vtk` instead of `tovtk`